fw/http.c

```diff
@@ -4746,8 +4746,10 @@ tfw_http_cli_error_resp_and_log(TfwHttpReq *req, int status, const char *msg,
 		nolog = tfw_blk_flags & TFW_BLK_ERR_NOLOG;
 	}
 
+	/* Do not log client port as it doesn't provide useful information
+	 * and could contain outdated cached data (see #1473). */
 	if (!nolog)
-		T_WARN_ADDR(msg, &req->conn->peer->addr, TFW_WITH_PORT);
+		T_WARN_ADDR(msg, &req->conn->peer->addr, TFW_NO_PORT);
```

Comment:
That's not the only place of printing with the client port, e.g. we also have similar printing in the warning at `tfw_http_req_fwd_send()`. There are also other macros with similar logic, e.g. `T_WARN_ADDR_STATUS()`. All the places must be reviewed whether it makes sense to print the port (e.g. there is an error message when we can't open listening port - we definitely need port in the message) and fix all the places.